docs/src/componentDocs/MobileStepper/markdown/MobileStepperExamples.mdx

```diff
@@ -1,86 +1,18 @@
-import Box from '@mui/material/Box';
-import MobileStepperDotVariant from '../images/mobileStepper_dot.png';
-import MobileStepperTextVariant from '../images/mobileStepper_text.png';
-import MobileStepperProgressVariant from '../images/mobileStepper_progress.png';
+import {MobileStepperDotVariant} from '../examples/MobileStepperDotVariant'
+import {MobileStepperTextVariant} from '../examples/MobileStepperTextVariant'
+import {MobileStepperProgressVariant} from '../examples/MobileStepperProgressVariant'
 
 ## Mobile Stepper
 
-The basic mobile stepper uses dots to indicate progress.
+<MobileStepperDotVariant />
 
-<Box
-  sx={{
-    width: '100%',
-    textAlign: 'center',
-  }}>
-  <Box
-    sx={{
-      width: '100%',
-      maxWidth: '400px',
-    }}
-    component="img"
-    alt="Mobile Stepper Dot Variant"
-    src={MobileStepperDotVariant}
-  />
-</Box>
-
-```
-<MobileStepper
-  activeStep={activeStep}
-  steps={steps}
-  leftButton={<BackButton />}
-  rightButton={<NextButton />}
-  variant={'dots'}
-/>;
-```
 
 ## Using Text / Progress Bar
```

Comment:
Follow the convention:

```
## Heading

Description Text
<Example />
```

So, there will be 2 separate examples:

```
## Using Text
Description of text
<ExampleText />

## Using Progress Bar
Description of progress bar
<ExampleProgress />

Reply:
Changes done as per suggestion